src/Microsoft.ComponentDetection.Orchestrator/Services/DetectorProcessingService.cs

```diff
@@ -109,7 +109,7 @@ public async Task<DetectorProcessingResult> ProcessDetectorsAsync(IDetectionArgu
                     exitCode = resultCode;
                 }
 
-                this.experimentService.RecordDetectorRun(detector, detectedComponents);
+                this.experimentService.RecordDetectorRun(detector, componentRecorder, detectionArguments);
```

Comment:
This fixes this exception: 

```
[12:43:34 DBG] The path: /home/jamie/scratch/dotnet/project-assets/project-assets.csproj could not be resolved relative to the root file:///D:/a/component-detection/component-detection/test/Microsoft.ComponentDetection.VerificationTests/resources/
System.UriFormatException: Invalid URI: The format of the URI could not be determined.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind, UriCreationOptions& creationOptions)
   at System.Uri..ctor(String uriString)
   at Microsoft.ComponentDetection.Orchestrator.Services.GraphTranslation.DefaultGraphTranslationService.MakeFilePathsRelative(ILogger logger, DirectoryInfo rootDirectory, HashSet`1 filePaths) in D:\a\component-detection\component-detection\src\Microsoft.ComponentDetection.Orchestrator\Services\GraphTranslation\DefaultGraphTranslationService.cs:line 215
```

which only happens on Windows because the `URI` class does not parse Linux paths on Windows:

https://github.com/microsoft/component-detection/blob/dae150a3b42770e7aadcf844f49a82d603b768d4/test/Microsoft.ComponentDetection.VerificationTests/resources/nuget/project-assets/project.assets.json#L4608-L4612

Tested on Windows and Ubuntu and doesn't seem to cause issues.